enhancements/update/duration-of-migration-to-multi-arch.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Duration of Migration To Multi Arch
+authors:
+  - "@hongkailiu"
+reviewers:
+  - "@wking"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
```

Comment:
Should @PratikMahajan be in the root `OWNERS` now that he's the updates team lead?  And then be the approver for this enhancement?

Reply:
https://github.com/openshift/enhancements/pull/1703